src/app/services/actions/query-action-creator-util.ts

```diff
@@ -103,9 +103,13 @@ const makeRequest = (httpVerb: string, scopes: string[]): Function => {
 
     if (query.sampleHeaders) {
       query.sampleHeaders.forEach(header => {
-        if (header.name !== '' && header.value !== '') {
-          sampleHeaders[header.name] = header.value;
+        if (header.name.toLowerCase() === 'content-type') {
+          return;
         }
+        if (header.name === '' && header.value === '') {
```

Comment:
No use in setting a header if either value is empty string. I think this should be an **or** condition.

Reply:
I actually think this check is redundant now